Delete_User.php

```diff
@@ -1,56 +1,73 @@
+<?php
```

Comment:
I add a function check if they have create a session(login). It will deliver to the login page if they wasn't login but want to access to some link that they should login first.